spec/factories/works.rb

```diff
@@ -10,7 +10,6 @@
     created_edtf { '1900' }
     abstract { 'test abstract' }
     citation { 'test citation' }
-    access { 'stanford' }
```

Comment:
why does this go away?

Reply:
I removed this so I could test that the DB (vs. this factory) was the actor responsible for setting the default value.